api_products.md

```diff
@@ -327,7 +327,7 @@ All parameters are optional.
 | Name  | Description |
 | ------------- | ------------- |
 | limit | Sets a limit of products returned. Must be a positive number. |
-| q | Passes a query string. Will do a full text search in the product name and description and will perform a prefix search with the product SKU. All products found by either search will be returned. |
+| q | Passes a query string. Will do a full text search in the product name and description and will perform a prefix search with the product SKU, scannable codes and weigh item codes. All products found by either search will be returned. |
```

Comment:
In der API heißen die Felder noch `weighedItemIds` und `eans`. Das sollte konsistent sein.